### PR TITLE
New Chinese localizations

### PR DESCRIPTION
**Pull request by hjaltisan**

### CODE REVIEW

File: src/sdk/translations/ifw_zh_CN.ts [hunk 1]
```diff
@@ -878,188 +878,188 @@
...
+        <translation>无法移除目录"%1"：%2</translation>
     </message>
 </context>
 <context>
     <name>QInstaller::CreateShortcutOperation</name>
     <message>
         <source>&lt;target&gt; &lt;link location&gt; [target arguments] [&quot;workingDirectory=...&quot;] [&quot;iconPath=...&quot;] [&quot;iconId=...&quot;] [&quot;description=...&quot;]</source>
-        <translation>&lt;目标&gt; &lt;链接位置&gt; [target arguments] [&quot;workingDirectory=...&quot;] [&quot;iconPath=...&quot;] [&quot;iconId=...&quot;] [&quot;description=...&quot;]</translation>
+        <translation>&lt;target&gt; &lt;link location&gt; [target arguments] ["workingDirectory=..."] ["iconPath=..."] ["iconId=..."] ["description=..."]</translation>
     </message>
```
Comment (BaldvinTh) [inline src/sdk/translations/ifw_zh_CN.ts:1051]: Translations got removed?
`target`, `link location`

File: src/sdk/translations/ifw_zh_CN.ts [hunk 2]
```diff
@@ -1286,65 +1285,65 @@ Error while loading %2</source>
...
-        <translation>写入设置失败。</translation>
+        <translation>未能写入设置。</translation>
     </message>
 </context>
 <context>
     <name>QInstaller::InstallIconsOperation</name>
     <message>
         <source>&lt;source path&gt; [vendor prefix]</source>
-        <translation>&lt;源路径&gt; [vendor prefix]</translation>
+        <translation>&lt;source path&gt; [vendor prefix]</translation>
```
Comment (BaldvinTh) [inline src/sdk/translations/ifw_zh_CN.ts:1295]: Translation missing

File: src/sdk/translations/ifw_zh_CN.ts [hunk 3]
```diff
@@ -1909,97 +1897,97 @@ Please copy the installer to a local drive</source>
...
-        <translation>安装程序将使用 %1 的磁盘空间。</translation>
+        <translation>安装程序将占用%1的磁盘空间。</translation>
     </message>
 </context>
 <context>
     <name>QInstaller::RegisterFileTypeOperation</name>
     <message>
         <source>&lt;extension&gt; &lt;command&gt; [description [contentType [icon]]]</source>
-        <translation>&lt;扩展名&gt; &lt;命令&gt; [description [contentType [icon]]]</translation>
+        <translation>&lt;extension&gt; &lt;command&gt; [description [contentType [icon]]]</translation>
```
Comment (BaldvinTh) [inline src/sdk/translations/ifw_zh_CN.ts:1931]: Missing translation?

**Review by BaldvinTh — COMMENTED** [review]
Some translations seem to have reverted to English? 😱

File: src/sdk/translations/ifw_zh_CN.ts (hunk 1 above)
Comment (hjaltisan) [reply src/sdk/translations/ifw_zh_CN.ts:1051]: These translations were removed on purpose by the translators

**Review by BaldvinTh — APPROVED** [review]
Translations being reverted was intentional by the translators

File: src/sdk/translations/ifw_zh_CN.ts (hunk 2 above)
Comment (hjaltisan) [reply src/sdk/translations/ifw_zh_CN.ts:1295]: These translations were removed on purpose by the translators

**Pull request merged**